internal/orchestrator/orchestrator.go

```diff
@@ -244,6 +244,11 @@ func (or *orchestrator) Init() (err error) {
 	return err
 }
 
+func Purge(ctx context.Context, ns *core.Namespace, plugins *Plugins, dxNodeName string) {
```

Comment:
Any reason to not put this on the `*orchestrator` (I think it should be non-nil and have access to namespace/plugins/node name at the time this is needed, right?)